go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/snyk/go-application-framework
 
-go 1.21
+go 1.21.0
```

Comment:
can we potentially increase that to 1.22.x? I think LS already states that it uses 1.22.x, so bumping the go versions consistently would be nice (also for CLI).

Reply:
Oh cool! I wasn't sure about that but heck yeah I could use range over integer in the test.

Reply:
Done!